.circleci/config.yml

```diff
@@ -5,6 +5,8 @@ version: 2.0
 
 common: &common
   working_directory: ~/repo
+  environment:
+    SOLC_VERSION: v0.4.23
```

Comment:
hardcoded for now - but will make dynamic down the road to support multiple versions